src/licensedcode/data/licenses/apache-exception-llvm.yml

```diff
@@ -0,0 +1,7 @@
+ +key: apache-exception-llvm
```

Comment:
IMHO we may need a better name.
@DennisClark what would this be? `llvm-apache-exception`

Reply:
@SaravananOffl All your keys are prefixed with a `+` .... invalid YAML!

Reply:
@pombredanne should i change the name also ?

Reply:
@SaravananOffl you can fix the file content, but keep the name for now until we hear back from @DennisClark